dapps/digital-twin-detail/src/components/overview/DetailOverviewTransactions.ts

```diff
@@ -40,13 +40,12 @@ export default class DetailOverviewTransactionsComponent extends mixins(EvanComp
       label: this.$t('_twin-detail.overview.date'),
       thClass: 'th-date',
     },
-    {
-      key: 'action',
-      label: '',
-      thClass: 'th-icon',
-    },
   ];
 
+  onRowClicked(row: TwinTransaction): void {
```

Comment:
wouldn't it be possible to put this functionality into an a-tag in the template?